docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+s# HVAC_SystemZoneAssignment - Rule 11-10
+
+**Schema Version:** 0.0.37  
+**Mandatory Rule:** True  
+**Rule ID:** 11-10  
+
+**Rule Description:** "The service water heating system type in the baseline building design shall match the minimum efficiency requirements in Section 7.4.2"  
+
+**Rule Assertion:** Options are PASS/FAIL/NOT_APPLICABLE/UNDETERMINED  
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, a + b  
+
+**Evaluation Context:** Each ServiceWaterHeatingEquipment  
+**Data Lookup:**   
+**Function Call:**
+
+1. get_SHW_equipment_associated_with_each_SWH_bat()
+
+**Applicability Checks:**
+- Each SHW Equipment is applicable
+- use get_SHW_equipment_associated_with_each_SWH_bat to get the SHW BATs and SHW use in the building: `shw_bats_and_equip_dict = get_SHW_equipment_associated_with_each_SWH_bat(B_RMD)`
+- look through each of the SHW bats in the building: `for shw_bat in shw_bats_and_equip_dict:`
+  - look through each SHW Equipment: `for service_water_heating_equipment_id in shw_bats_and_equip_dict[shw_bat]: CONTINUE TO RULE LOGIC`
+
+
+    ## Rule Logic:
+  - create a value to track the rule status: `rule_status = "FAIL"`
+  - create a variable to track any notes: `rule_note = """`
+  - we need to find the following information about the SWH:
+    - swh_type - (ELECTRIC_RESISTANCE_INSTANTANEOUS, ELECTRIC_RESISTANCE_STORAGE, GAS_STORAGE, OTHER)
+    - swh_type - one of the SWH BATs
+    - input_capacity (btu/hr)
+    - storage_volume (gallons)
+    - draw_pattern (VERY_SMALL, LOW, MEDIUM, or HIGH)
+    - capacity_per_volume (btu/hr/gallon)
+  - get the service water heating system: `swh_eq = get_object_by_id(service_water_heating_equipment_id,RMD)`
+  - get the type of tank: `tank_type = swh_eq.tank.type`
+  - get the type of heater: `swh_type = swh_eq.heater_type`
+  - the type will be one of the ServiceWaterHeaterTankOptions, which are more detailed than the four types we need.  Determine our type based on the detailed type.
+  - set the swh_type to OTHER.  If the tank_type is an identifiable type, swh_type will be overwritten to that type: `swh_type = "OTHER"`
```

Comment:
It took me a few minutes to understand what you meant by saying the swh_type would be overwritten to the tank_type if it is identifiable. I think you are saying the default value of swh_type is OTHER and it will be replaced with something more appropriate if possible (as determined by the logic below)